### PR TITLE
Setup google form for sign up

### PR DESCRIPTION
**Pull request by edderrd**
link google form in `/index-new` buttons

### CODE REVIEW

**Comment by edderrd** [comment]
[Setup google form for sign up](https://trello.com/c/08iZ3O5f/43-setup-google-form-for-sign-up)

File: pages/index-new.js [hunk 1]
```diff
@@ -117,7 +118,7 @@ const Page = () => {
         <Public>
           <MenuBar>
             <Link href="/about">About</Link>
```
Comment (mckelvey) [inline pages/index-new.js:120]: Non-blocker: want to strip this link while here @edderrd? Seems like we don’t need it at present.

**Review by mckelvey — APPROVED** [review]
Niiiice!

File: pages/index-new.js (hunk 1 above)
Comment (edderrd) [reply pages/index-new.js]: striped @mckelvey

**Pull request merged**